pandas/tests/plotting/test_converter.py

```diff
@@ -76,7 +76,7 @@ def test_dont_register_by_default(self):
         call = [sys.executable, "-c", code]
         assert subprocess.check_call(call) == 0
 
-    @td.skip_if_no("matplotlib", min_version="3.1.3")
+    @td.skip_if_no("matplotlib")
```

Comment:
There is a `td.skip_if_no_mpl` decorator. seems logical to use that?

Reply:
I think we can get rid of them altogether, lets see if ci is green